src/services/authentication/AuthenticationService.ts

```diff
@@ -38,16 +38,25 @@ export class AuthenticationService {
   login(credentials: ICredentials): ng.IPromise<string> {
     this.setCredentials(credentials);
     let deferred: ng.IDeferred<string> = this.$q.defer();
-    this.dmrService.readAttribute({address: [], name: "launch-type"})
-      .then(response => {
-        this.setCredentials(credentials);
-        this.launchType.set(response);
-        this.availability.startApiAccessibleCheck();
-        deferred.resolve();
-      }, error => {
-        this.logout();
-        deferred.reject(error);
-      });
+    this.dmrService.readResource({
+      address: [],
+      recursive: true,
+      "include-runtime": true,
+      "recursive-depth": 2
+    }).then((response: any) => {
+      let hasJGroupsStack: boolean = true;
+      let launchType: string = response["launch-type"];
+      if (LaunchTypeService.STANDALONE_MODE === launchType) {
+        hasJGroupsStack = isNotNullOrUndefined(response["subsystem"]["datagrid-jgroups"]);
```

Comment:
let's change this to response.subsystem["datagrid-jgroups"] to keep the tslint checks happy